pages/pipelines/block_step.md.erb

```diff
@@ -253,6 +253,7 @@ _Optional attributes:_
     <td><code>required</code></td>
     <td>
       A boolean value that defines whether the field is required for form submission.<br>
+      When this value is set to <code>false</code>, non-multiple options will always displayed in a dropdown box, regardless of how many are provided.<br>
```

Comment:
I found this a bit difficult to articulate in a clear manner, and also was unsure as to how much detail to add, so keen for some advice on wording here :)

Reply:
Okay, let me check the logic first:

- if (can select multiple options) -> Checkboxes
- else if (options > 6 || !required) -> Dropdown
- else -> radio buttons

So, it only displays as radio buttons when you can only select one option AND you're required to select an option AND there are 6 or fewer options.

Is that right? If so, perhaps:

```suggestion
      When this value is set to <code>false</code> and users can only select one option, the options display in a dropdown menu, regardless of how many options there are.
<br>
```

Reply:
Yes that wording is much clearer, good call.

Reply:
And yes that logic is correct!

Reply:
Yay! I'll add this suggestion for the other page, too.